c++/src/kj/compat/http.h

```diff
@@ -572,6 +572,9 @@ class WebSocket {
   // if this WebSocket implementation is able to perform the pump in an optimized way, better than
   // the default implementation of pumpTo(). The default implementation of pumpTo() always tries
   // calling this first, and the default implementation of tryPumpFrom() always returns null.
+
+  virtual uint64_t sentBytesCount();
```

Comment:
Nitpick: Normally I'd use either a plural word or a "-Count" suffix but not both. So I'd make this `sentByteCount()` (non-plural "byte").

Reply:
Switched to `sentByteCount()`